packages/react/src/divider-with-text/docs/overview.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: Divider with Text
+description: A horizontal rule with text that separates blocks of content.
+group: Content
+storybookPath: /story/Content-DividerWithText--basic
+unreleased: true
+figmaGalleryNodeId: 18634%3A85108
+relatedComponents: ['divider']
+---
+
+```jsx live
+<DividerWithText>
+	<Text>Divider label</Text>
+</DividerWithText>
+```
+
+The divider with label should be used sparingly and used only where appropriate to give context on the content split by the divider. For example, this variant can be used for general sign in divider use.
+
+<DoHeading />
+
+- use to create visual space and separation between blocks of content like form field sets in the main content area
+- divide and logically group content sections to enhance clarity and organisation
+- position above buttons groups at the bottom of a form to give prominence to the action area
+- include a concise label.
```

Comment:
I think we should also add something to effect of "use the correct semantics on the text element"